programmers/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,9 @@
+function solution(s) {
+  let answer = "";
+  if (s.length % 2 == 0) {
+    answer = s.substr([s.length / 2 - 1], 2);
+  } else {
+    answer = s[parseInt(s.length / 2)];
+  }
+  return answer;
```

Comment:
함수 내에서 굳이 변수를 선언할 필요가 없는 이런 문제 같은 경우엔
바로 return 하는게 괜찮을 수도 있겠다는 생각도 드네요
```suggestion
  if (s.length % 2 == 0) {
    return s.substr([s.length / 2 - 1], 2);
  } else {
    return s[parseInt(s.length / 2)];
  }
```